pipelines/plans/Jenkinsfile

```diff
@@ -0,0 +1,60 @@
+email = "popu-servers+jenkins@populate.tools "
+pipeline {
+    agent any
+    environment {
+        PATH = "$HOME/.rbenv/shims:$PATH"
+        GOBIERTO_ETL_UTILS = "/var/www/gobierto-etl-utils/current/"
+        MATARO_ETL = "/var/www/gobierto-etl-mataro/current/"
+        WORKING_DIR = "/tmp/mataro_plans"
+        MATARO_ID = "8121"
+        // Variables that must be defined via Jenkins UI:
+        // GOBIERTO = "/var/www/gobierto/current"
+        // API_HOST = "https://mataro.gobify.net"
+        // PLAN_ID = "52"
+    }
+    options {
+        retry(3)
+    }
+    stages {
+        stage('Clean working dir') {
+          steps {
+              sh "rm -rf ${WORKING_DIR}"
+          }
+        }
+        stage('Extract > Download data sources - Plan data') {
```

Comment:
Because in this ETL you are going to add different plans, what about adding a prefix for each plan, so the stages would be:

- `[PAM] Extract > Download data sources...`
- `[PAM] Transform....`
- `[PAM] Load....`
- `[Urban wadus] Extract...`

Or any other prefix or way to distinguish them

Reply:
Added [PAM] and [Urban Agenda 2030]